### PR TITLE
QJsonObject pretty printer

### PR DESCRIPTION
**Pull request by kropp**

### CODE REVIEW

File: core.py [hunk 1]
```diff
@@ -713,7 +713,7 @@ class QVariantPrinter:
     def __init__(self, val):
         self.val = val
 
-    def children(self):
+    def tmp(self):
```
Comment (Lekensteyn) [inline core.py]: What does this do? The `to_string` change below seems a copy/paste, does it need some other form of transformation?

**Review by Lekensteyn — COMMENTED** [review]
Thanks for your PR! Have some questions though

**Comment by kropp** [comment]
Thanks! Fixed. Should have removed it before submitting.

The change is to render underlying value directly.

**Comment by Lekensteyn** [comment]
LGTM, thanks!

**Pull request merged**